design-doc-template.adoc

```diff
@@ -0,0 +1,39 @@
+= Title
+:author:            Your Name
+:email:             your.email@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
```

Comment:
I guess the bullet list is not correct here?

Reply:
I don't really know to be honest. I just know when I tried to generate the document with `asciidoctor` it had errors.